src/PNEventEngine/PNEventEngine.csproj

```diff
@@ -0,0 +1,176 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
this project is just for reference right? I mean we will eventually get rid of this!!!!

Reply:
That's fine to have it as of now. I just thought of confirming